CLI/turbot.py

```diff
@@ -172,16 +214,32 @@ def main():
             sys.exit(exit_code)
         sys.exit(0)
 
-    elif args.D == True:                        # Disassembles given binary
+    elif args.D:                        # Disassembles given binary
         print('Running Disassemble Only')
+        if args.destination[-1] == '/':
```

Comment:
Doesn't support specifying current directory with the '.' wildcard. (or '..' and such, for the matter)
Happens in multiple locations, but I'll only comment once